### ISSUE TITLE
Screenfulls of No Joy for 2 days running......

### ISSUE THREAD

**Issue by Printersmith** [mehtadone/PTFeeder#285]
It has not been quite 48 hours since launch, but I don't think this is normal.  The Bot is telling me that it cannot pull any pricing information from the exchange.  Everything "seems to be" working but I have not executed a single buy order yet.  
![no-joy-ever](https://user-images.githubusercontent.com/37848694/37991067-6254cb60-31c5-11e8-9e84-919681573816.png)

**Comment by mehtadone** [comment]
Can you try running the timesync tool on here: 
https://github.com/mehtadone/PTFeeder/wiki/Common-errors#3-sequence-contains-no-element--could-not-get-btc-price-change-from-all-exchanges-returning-a-massive-value-so-you-end-up-in-superbear-error

**Comment by Printersmith** [comment]
That's helping for sure, thanks!  I don't get why these servers are so wacked out with Time zones.  Mine is supposedly in California, but was pegged 8 hours ahead of PST.  I kept changing the time and it would auto-reset back to stupid time, rather than real time.  I wrote a nice letter to Support :-)

So now its trading on Binance, but nothing on Bittrex.  I am supposed to be able to run up to 4 instances on the same server, correct??  I have both locations in the host file, and it's writing pair files for both.....

I am going to add some more BTC to bittrex and keep studying the settings.  I am on 1.4.latest and my .json file still shows "cost" rather than "maxcost" so I changed that.
